### PR TITLE
Rework transports tables

### PR DESCRIPTION
**Pull request by pablogs9**
Before merge:
- [ ] https://github.com/micro-ROS/micro-ROS.github.io/pull/241

### CODE REVIEW

File: _docs/overview/transports/index.md [hunk 1]
```diff
@@ -5,10 +5,27 @@ permalink: /docs/overview/transports/
 
 micro-ROS uses the resource-optimized [DDS for Extremely Resource Constrained Environments (DDS-XRCE) standard](https://www.omg.org/spec/DDS-XRCE/), implemented by [eProsima's Micro-XRCE-DDS](https://github.com/eProsima/Micro-XRCE-DDS/).
 
-It supports are variety of data link and transport protocols. However, the support depends on the underlying RTOS, as specified in the following table.
+It supports are variety of data link and transport protocols. However, the support depends on the underlying RTOS and the selected hardware.
+
+The following table specifies the available transports for the **officially supported** boards and RTOS. You can find detailed information about hardware support [here](/docs/overview/hardware/).
```
Comment (ralph-lange) [inline _docs/overview/transports/index.md:10]:
```suggestion
The following table specifies the available transports for the Reference and Tier-2 micro-ROS boards for each officially supported RTOS. You can find detailed information about hardware support [here](/docs/overview/hardware/).
```

**Review by ralph-lange — CHANGES_REQUESTED** [review]

File: _docs/overview/hardware/index.md [hunk 2]
```diff
@@ -3,114 +3,311 @@ title: Supported Hardware
...
+        <b>Resources:</b>
+        <ul>
+            <li><a href="https://www.pjrc.com/store/teensy40.html">Official website</a></li>
+        </ul>
+        <i> This board is supported under <a href="https://github.com/micro-ROS/micro_ros_arduino">micro-ROS for Arduino</a></i>    
+    </div>
+  </div>
+
+  <div class="hardwareitem_image">
+    <img src="imgs/teensy41.jpg">
```
[the remark this answers is not in the document]
Comment (pablogs9) [reply _docs/overview/hardware/index.md:146]: Done

File: _docs/overview/hardware/index.md [hunk 3]
```diff
@@ -3,114 +3,311 @@ title: Supported Hardware
...
+    </div>
+  </div>
+
+  <div class="hardwareitem_image">
+    <img src="https://www.olimex.com/Products/ARM/ST/STM32-E407/images/STM32-E407-02.jpg">
+  </div>
+</div>
+
+## Tier 2 micro-ROS boards
+
```
[the remark this answers is not in the document]
Comment (pablogs9) [reply _docs/overview/hardware/index.md:75]: Done

File: _docs/overview/hardware/index.md [hunk 4]
```diff
@@ -3,114 +3,311 @@ title: Supported Hardware
...
+  </div>
+
+  <div class="hardwareitem_image">
+    <img src="imgs/3.jpg">
+  </div>
+</div>
+
+
+## Community supported boards
+
```
[the remark this answers is not in the document]
Comment (pablogs9) [reply _docs/overview/hardware/index.md:224]: Done

File: _docs/overview/hardware/index.md [hunk 5]
```diff
@@ -3,114 +3,311 @@ title: Supported Hardware
...
+The micro-ROS hardware support is divided in three categories: 
+- reference micro-ROS board, 
+- tier 2 boards and, 
+- community supported boards.
+
+*In order to check the most recent hardware support visit the [micro_ros_setup repo](https://github.com/micro-ROS/micro_ros_setup)*
+
+## Reference micro-ROS board
+
```
[the remark this answers is not in the document]
Comment (pablogs9) [reply _docs/overview/hardware/index.md:45]: Done

**Review by ralph-lange — DISMISSED** [review]

File: _docs/overview/transports/index.md [hunk 6]
```diff
@@ -5,10 +5,26 @@ permalink: /docs/overview/transports/
 
 micro-ROS uses the resource-optimized [DDS for Extremely Resource Constrained Environments (DDS-XRCE) standard](https://www.omg.org/spec/DDS-XRCE/), implemented by [eProsima's Micro-XRCE-DDS](https://github.com/eProsima/Micro-XRCE-DDS/).
 
-It supports are variety of data link and transport protocols. However, the support depends on the underlying RTOS, as specified in the following table.
+It supports are variety of data link and transport protocols. However, the support depends on the underlying RTOS and the selected hardware.
```
Comment (FranFin) [inline _docs/overview/transports/index.md:8]:
```suggestion
It supports a variety of data link and transport protocols. However, the support depends on the underlying RTOS and selected hardware.
```

**Review by FranFin — CHANGES_REQUESTED** [review]

**Review by jamoralp — APPROVED** [review]

**Review by FranFin — APPROVED** [review]

**Review by ralph-lange — DISMISSED** [review]

**Review by FranFin — APPROVED** [review]

**Pull request merged**